wazo_calld/plugins/calls/bus_consume.py

```diff
@@ -277,7 +278,13 @@ def _users_services_dnd_updated(self, event):
         user_uuid = event['user_uuid']
         enabled = event['enabled']
         interface = f'Local/{user_uuid}@usersharedlines'
-        if enabled:
-            ami.pause_queue_member(self.ami, interface)
-        else:
-            ami.unpause_queue_member(self.ami, interface)
+        try:
+            if enabled:
+                ami.pause_queue_member(self.ami, interface)
+            else:
+                ami.unpause_queue_member(self.ami, interface)
+        except WazoAmidError as e:
+            if e.details['original_error'] == 'Interface not found':
```

Comment:
Is there a better way to test this?

Reply:
It's hard to do a good test. The bug is that log files get filled with thousands of tracebacks but I think there are now behavior change with or without handling the error